packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -197,6 +209,20 @@ async function verifyOIDCConfig(config: OIDCConfigs) {
   await verifySSOConfig(ConfigType.OIDC, config.configs[0])
 }
 
+export async function verifyAIConfig(
+  config: AIConfig,
+  existingConfig?: AIConfig
+) {
+  if (!existingConfig) return
```

Comment:
This is a bit weird. A caller could do `verifyAIConfig(config)` without realising it would never do anything at all. If `existingConfig` is required, it make it required as a type and throw an error if it's not provided. Think it makes more sense for this responsibility to be on the caller.